account.go

```diff
@@ -812,16 +820,27 @@ type AccountTOSAcceptanceParams struct {
 
 // Returns a list of accounts connected to your platform via [Connect](https://stripe.com/docs/connect). If you're not a platform, the list is empty.
 type AccountListParams struct {
-	ListParams   `form:"*"`
+	ListParams `form:"*"`
+	// Specifies which fields in the response should be expanded.
+	Expand []*string `form:"expand"`
+
 	Created      *int64            `form:"created"`
 	CreatedRange *RangeQueryParams `form:"created"`
 }
 
+// AddExpand appends a new field to expand.
```

Comment:
Yes, to shadow the one from the embedded params.

Reply:
But why?

Reply:
We shouldn't break years of integrations, having both is really a better experience for everyone no?

Reply:
If we omitted the per-param-struct `.AddExpand` then it would fall back to the embedded param `.AddExpand` and you would get a weird situation where

```go
p := FooParams{}
p.AddExpand("bar")
len(p.Expand) // would equal 0
```
Existing users are using `.AddExpand` already. We are doing this in a major so this sort of breakage we could probably get away with, but I think it's confusing and would prefer to avoid.

Also IMO the main reason we are doing this for `Expand` is for consistency with `Metadata`, so we might as well be completely consistent.

Reply:
I changed my mind. I deprecated `.AddExpand` and that should be enough.

```
p.AddExpand("bar")
len(p.Expand)
```
doesn't seem like a very likely possibility

Reply:
(`.AddExpand` will still work for adding things to the expand array, though, so for Remi's concern we won't be breaking years of integrations)